README.md

```diff
@@ -46,6 +47,80 @@ npx truffle compile
 npm run networks-inject 
 ```
 
+# Deploying a simple market maker scenario to Rinkeby:
+
+The following script deploys a simple market maker order and a necessary owl order, to enable trading:
+```
```

Comment:
nit: if you add an sh after ``` you will get shell syntax highlighting.

Reply:
I think in this case `bash` would also give similar highlighting... not sure if one or the other looks better.